graph-db/src/utils/bootstrapGremlinServer.js

```diff
@@ -0,0 +1,54 @@
+const { initializeGremlinConnection } = require('./gremlin/initializeGremlinConnection')
+const { indexCmrCollection } = require('./indexing/indexCmrCollection')
+const { fetchPageFromCMR } = require('./cmr/fetchPageFromCmr')
+const { clearScrollSession } = require('./cmr/clearScrollSession')
+const { getEchoToken } = require('./cmr/getEchoToken')
+
+/**
+ * Harvests collections from CMR and loads them into graph db
+ * @returns {Array} Errors from indexing into graph db
+ */
+exports.bootstrapGremilinServer = async () => {
+  const echoToken = await getEchoToken()
+  const gremlin = initializeGremlinConnection()
+  const response = await fetchPageFromCMR(null, echoToken)
+  const results = (await response.json()).items
+  const scrollId = response.headers.get('CMR-Scroll-Id')
+  const partitionedSearchResults = []
```

Comment:
I don't think you need this variable anymore, you push to it two times below but don't do anything with it.